autonomy/analyse/service.py

```diff
@@ -36,15 +43,101 @@
 from autonomy.configurations.loader import load_service_config
 
 
-REQUIRED_SETUP_PARAMETERES = (
-    "safe_contract_address",
-    "all_participants",
-)
+ABCI_CONNECTION_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "config": {
+            "type": "object",
+            "required": [
+                "host",
+                "port",
+                "use_tendermint",
+            ],
+        },
+        "required": ["config"],
+    },
+}
 
-REQUIRED_PARAM_VALUES = (
-    "share_tm_config_on_startup",
-    "on_chain_service_id",
-)
+LEDGER_CONNECTION_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "config": {
+            "type": "object",
+            "properties": {
+                "ledger_apis": {
+                    "type": "object",
+                    "properties": {
+                        "ethereum": {
+                            "type": "object",
+                            "properties": {
+                                "address": {},
+                                "chain_id": {},
+                                "poa_chain": {},
+                                "default_gas_price_strategy": {},
+                            },
+                            "required": [
+                                "address",
+                                "chain_id",
+                                "poa_chain",
+                                "default_gas_price_strategy",
+                            ],
+                        },
+                    },
+                    "required": ["ethereum"],
+                }
+            },
+            "required": ["ledger_apis"],
+        },
+    },
+    "required": ["config"],
+}
```

Comment:
This is not generic. Very much scoped to ethereum version of ledger connection..

Reply:
Addressed https://github.com/valory-xyz/open-autonomy/pull/1807/commits/477b3cc4bd7edc1563faf009b198ac2cd5475f1f#diff-739012fc1855ea8228dad566ed5f58aaf94767441f8391c8962368907ca155b2R70